src/components/FacilityListItem.tsx

```diff
@@ -1,39 +1,65 @@
-import { TableRowType } from '@common/types/gristData'
+import { GristLabelType } from '@common/types/gristData'
 import classNames from '@lib/classNames'
 import { useDistanceToUser } from '@lib/hooks/useDistanceToUser'
 import { useIsFacilityOpened } from '@lib/hooks/useIsFacilityOpened'
+import { useLabels } from '@lib/LabelsContext'
+import { MinimalRecordType } from '@lib/mapRecordToMinimum'
 import { useTexts } from '@lib/TextsContext'
 import Link from 'next/link'
 import { FC } from 'react'
 
-interface FacilityListItemPropsType extends TableRowType {
+interface FacilityListItemPropsType extends MinimalRecordType {
   className?: string
 }
 
 export const FacilityListItem: FC<FacilityListItemPropsType> = ({
   className = ``,
-  id,
-  fields,
+  ...record
 }) => {
+  const { id, title, latitude, longitude, labels } = record
   const texts = useTexts()
   const distance = useDistanceToUser({
-    latitude: fields.lat,
-    longitude: fields.long2,
+    latitude,
+    longitude,
   })
-  const isOpened = useIsFacilityOpened(fields)
+  const isOpened = useIsFacilityOpened(record)
+  const labelsWithData = useLabels()
+  const recordLabels = labels
+    .map((lId) => labelsWithData.find((l) => l.id === lId))
+    .filter(Boolean) as GristLabelType[]
+  const semanticLabels = recordLabels.filter(
```

Comment:
I wonder if we can name this a bit clearer? How about `topicLabels` or `issueLabels`?

Reply:
I agree that topicLabels is clearer! 👍  Not too sure about issue though. Is being a family or a child an issue? 😛

Reply:
I renamed `semanticLabels` to `topicsLabels` and `targetLabels` to `targetAudienceLabels`